drivers/flash/Kconfig.nor

```diff
@@ -75,4 +75,13 @@ config SPI_NOR_IDLE_IN_DPD
 	  long periods, and when used the impact of waiting for mode
 	  enter and exit delays is acceptable.
 
+config SPI_NOR_HIGH_PERF_MODE
```

Comment:
This Kconfig options should be clearly named to indicate that it turns on vendor/chip specific behaviour.
Same with description for the option.